internal/assets/opencode_review_transport_plugin_test.go

```
@@ -246,6 +262,242 @@ printf '%s\n' "$complete" >> "$GENTLE_AI_RELAY_LOG"
+	const harness = `import plugin from "./plugin.mts"
+const hooks = await plugin({ directory: process.cwd(), worktree: process.cwd() })
+const before = { args: { subagent_type: "review-risk", prompt: "Go must receive this original host prompt" } }
+let refused = ""
+try {
+  await hooks["tool.execute.before"]({ tool: "task", sessionID: "session", callID: "call" }, before)
+} catch (cause) {
+  refused = cause instanceof Error ? cause.message : String(cause)
+}
+console.log(JSON.stringify({ prompt: before.args.prompt, refused }))
+`
+	output, log, _ := runOpenCodeTransportPluginHarness(t, map[string]string{"plugin.mts": string(source)}, harness, "", "")
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)